ui-lib/components/Button/index.js

```diff
@@ -0,0 +1,131 @@
+// @flow
+import React from "react";
+import styled, { css } from "styled-components";
+import { fontSize, color } from "styled-system";
+import type { ThemedComponent } from "@ui/styles/StyleProvider";
+import fontFamily from "@ui/styles/styled/fontFamily";
+import { fontSizes } from "@ui/styles/theme";
+
+type Props = {
+  Icon?: React$ComponentType<*>,
+  children: React$Node,
+  onClick: () => void,
+  ff?: string,
+  color?: string,
+  fontSize?: number,
+  type?: "primary" | "secondary",
+  iconPosition?: "right" | "left",
+};
+
+const IconContainer: ThemedComponent<{ iconPosition: "right" | "left" }> = styled.div`
+  display: inline-block;
+  ${p => (p.iconPosition === "right" ? `margin-right: 10px;` : `margin-left: 10px;`)}
```

Comment:
I think that the margins are mixed up. (right -> right / left -> left instead of right -> left / left -> right)